dev-client/src/screens/LoginScreen.tsx

```diff
@@ -125,22 +128,26 @@ export const LoginScreen = () => {
       </Column>
       <Box flexGrow={3} />
       <Column pb="60px" alignItems="center" justifyContent="flex-end">
-        <TerrasoLogo width="122px" height="39px" />
-        <Box h="12px" />
-        <Text variant="caption" color="primary.contrast">
-          {t('login.description')}
-        </Text>
-        {isDevelopmentMode && (
-          <Text variant="caption" color="primary.contrast">
-            {APP_CONFIG.appleClientId}
+        <Row alignItems="flex-end">
+          <Text variant="body1" color="primary.contrast" mr="5px" mb="6px">
+            {t('login.from')}
           </Text>
-        )}
+          <TerrasoLogo width="122px" height="39px" />
+        </Row>
       </Column>
     </Column>
   );
 };
 
 const styles = StyleSheet.create({
+  loginButton: {
+    backgroundColor: 'white',
```

Comment:
This doesn't work with `background.default` or `primary.contrast`. Would like to understand why/fix.